release-notes.md

```diff
@@ -366,6 +374,70 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v46.0
+
+This release has been tracked in [v46.0
+group](https://github.com/orgs/cloud-hypervisor/projects/6/views/4?filterQuery=release%3A%22Release+46%22)
+of our [roadmap project](https://github.com/orgs/cloud-hypervisor/projects/6/).
+
+### Advisory Locks Support for Disk Image Files
+
+Now advisory locks support is added and enabled by default to prevent
+the same disk image file (that backs up a `virtio-blk` device) from being
+written by multiple Cloud Hypervisor processes. (#6974)
+
+### Improved Error Reporting with VM Resizing
+
+Instead of returning a generic error `400` (e.g. `BadRequest`), users
+now get a more specific error `429` (e.g. `TooManyRequests`) when a
+pending VM resizing is not completed. This allows users to better handle
+different errors, say retrying the request when applicable. (#7043)
+
+### IPv6 Address Support
+
+Now the TAP interface being automatically created along with a
+`virtio-net` device accepts IPv6 addresses. (#7048)
```

Comment:
Ditto here - "It is now possible to specify an IPv6 address and mask when creating a network interface with `--net`. etc.